rdt/hyper_transformer.py

```diff
@@ -73,6 +75,42 @@ class HyperTransformer:
         'b': 'boolean',
         'M': 'datetime',
     }
+    DEFAULT_TRANSFORMERS = {
+        'numerical': NumericalTransformer,
+        'integer': NumericalTransformer(dtype=int),
+        'float': NumericalTransformer(dtype=float),
+        'categorical': CategoricalTransformer(fuzzy=True),
+        'boolean': BooleanTransformer,
+        'datetime': DatetimeTransformer,
+    }
+    _DTYPES_TO_DATA_TYPES = {
+        'i': 'integer',
+        'f': 'float',
+        'O': 'categorical',
+        'b': 'boolean',
+        'M': 'datetime',
+    }
+
+    @staticmethod
+    def get_transformers_by_type():
+        """Build a ``dict`` mapping data types to valid existing transformers for that type.
+
+        Returns:
+                dict:
+                    Mapping of data types to a list of existing transformers that take that
+                    type as an input.
+        """
+        data_type_transformers = {}
+        transformer_classes = inspect.getmembers(sys.modules[__name__], inspect.isclass)
```

Comment:
Rather than using `inspect`, I would navigate the subclasses of `BaseTransformer` by adding a `get_subclasses` classmethod to it.

Something similar is done in SDGym: https://github.com/sdv-dev/SDGym/blob/79321b57a1b2dd416426fc9088e1ff771dd82e9c/sdgym/synthesizers/base.py#L17

Also, as a note, when implementing it here I would merge the `get_subclasses` and `get_baselines` logic, so only the classes that do not inherit from `ABC` directly are included in the output. By doing it this way, we can skip any intermediate abstract Transformers that may end up defining.